src/app/campaign/flight/flight-form-control-container.component.ts

```diff
@@ -121,7 +146,7 @@ export class FlightFormControlContainerComponent implements OnInit, OnDestroy {
 
   // updates the form from @Input() set flight
   setFlightForm(flight: Flight) {
-    // reset the form
```

Comment:
if we use `reset` it also resets the controls to pristine, which doesn't seem like something we want on every update cycle